app/src/main/java/fr/free/nrw/commons/review/ReviewInterface.java

```diff
@@ -10,8 +10,8 @@
  * Interface class for peer review calls
  */
 public interface ReviewInterface {
-    @GET("w/api.php?action=query&format=json&formatversion=2&list=recentchanges&rcprop=title|ids&rctype=new|log&rctoponly=1&rcnamespace=6&rctag=android%20app%20edit")
-    Observable<MwQueryResponse> getRecentChanges(@Query("rcstart") String rcStart);
+    @GET("w/api.php?action=query&format=json&list=recentchanges&continue=-||&formatversion=2&rcnamespace=6&rcprop=title|ids&rctype=new|log&rctoponly=1")
```

Comment:
Sorry I just noticed that now all recent uploads are shown.
In fact, the app's review should only show uploads that have been uploaded **by the app**, that's the role of `&rctag=android%20app%20edit`. Would you mind re-introducing this parameter? Thanks!

Reply:
I am sorry for missing that out. I'll reintroduce it.

Reply:
I have added it but I doubt if it is fetching the recent changes now (no changes between 27th February and 9th March):
![api sandbox](https://user-images.githubusercontent.com/83745993/224234483-73ba442a-4210-462f-acff-d153046857d1.png)